test/dummy/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_01_09_132519) do
+ActiveRecord::Schema.define(version: 2020_02_11_145825) do
```

Comment:
Whoops. @casperisfine . Not sure why these changes weren't removed in https://github.com/powerhome/shipit-engine/commit/a210081966d860400107e216f80446ab2f1f366a

Reply:
I should have seen it. No big deal.

Reply:
My apologies.

Reply:
No worries. I'll fix that on master.

Reply:
You're a gentle-person and a scholar.